store/store.go

```diff
@@ -75,13 +75,16 @@ func NewInMemory(ti TraceID) *InMemory {
 	}
 }
 
+
 func New(cfg config.Store, traceID TraceID) (MsgStore, error) {
 	if cfg.Type == "redis" {
 		ms, err := NewRedis(cfg.RedisHost, cfg.RunID, traceID)
 		if err != nil {
 			return nil, err
 		}
 		return ms, nil
```

Comment:
can we infer this from existing config, 

`TotalMessages!=-1 && consumer.Enabled`